src/views/SystemIntake/Page1/index.tsx

```diff
@@ -0,0 +1,218 @@
+        label="Same as Requestor"
+        name="isBusinessOwnerSameAsRequestor"
+        onChange={(e: any) => {
+          if (e.target.checked) {
+            setSameness(true);
+            formikProps.setFieldValue(
+              'businessOwner',
+              formikProps.values.requestor
+            );
+            formikProps.setFieldValue(
+              'businessOwnerComponent',
+              formikProps.values.requestorComponent
+            );
+          } else {
+            setSameness(false);
+          }
+        }}
+        value=""
+      />
+      <Field
+        as={TextField}
+        disabled={isReqAndBusOwnerSame}
+        id="IntakeForm-BusinessOwner"
+        maxLength={50}
+        name="businessOwner"
+      />
+      <Field
+        as={DropdownField}
+        disabled={isReqAndBusOwnerSame}
+        id="IntakeForm-BusinessOwnerComponent"
+        label="Business Owner Component"
+        name="businessOwnerComponent"
+      >
+        <Field as={DropdownItem} name="Select an option" value="" />
+        {cmsDivisionsAndOffices.map(office => (
+          <Field
+            as={DropdownItem}
+            key={`BusinessOwnerComponent-${office.acronym}`}
+            name={office.name}
+            value={office.name}
+          />
+        ))}
+      </Field>
+      <Field
+        as={TextField}
+        id="IntakeForm-ProductManager"
+        label="CMS Project/Product Manager, or lead"
+        maxLength={50}
+        name="productManager"
+      />
+      <Field
+        as={DropdownField}
+        id="IntakeForm-ProductManagerComponent"
+        label="Product Manager Component"
+        name="productManagerComponent"
+      >
+        <Field as={DropdownItem} name="Select an option" value="" />
+        {cmsDivisionsAndOffices.map((office: any) => (
+          <Field
+            as={DropdownItem}
+            key={`ProductManagerComponent-${office.acronym}`}
+            name={office.name}
+            value={office.name}
+          />
+        ))}
+      </Field>
+
+      <fieldset className="usa-fieldset margin-top-3">
+        <legend className="usa-label margin-bottom-1">
+          Is your project team currently collaborating/consulting with anyone of
+          the following?
+        </legend>
+        <FieldArray name="governanceTeams">
+          {arrayHelpers => (
+            <>
+              {cmsGovernanceTeams.map(team => {
+                const kebabValue = team.value.split(' ').join('-');
+                return (
+                  <CheckboxField
+                    key={team.value}
+                    checked={formikProps.values.governanceTeams.includes(
+                      team.value
+                    )}
+                    id={`governanceTeam-${kebabValue}`}
+                    label={team.name}
+                    name="governanceTeam"
+                    onBlur={() => {}}
+                    onChange={e => {
+                      if (e.target.checked) {
```

Comment:
Here is some wild logic that determines whether checkboxes should be checked.

Since there is a `None` option in these checkboxes, we need to uncheck all other options if `None` is checked.
On the flip side, we need to uncheck `None` if one of the checkboxes are checked.

This design is still being iterated on and is subject to change.